packages/release/label-data-alt.service

```diff
@@ -0,0 +1,17 @@
+[Unit]
+Description=Label alternative data partition
+DefaultDependencies=no
+Conflicts=shutdown.target
+# This is the partition GUID for the 'BOTTLEROCKET-DATA-ALT' partition
+Wants=dev-disk-by\x2dpartuuid-69040874\x2d417d\x2d4e26\x2da764\x2d7885f22007ea.device
+After=dev-disk-by\x2dpartuuid-69040874\x2d417d\x2d4e26\x2da764\x2d7885f22007ea.device
+
+[Service]
+Type=oneshot
+
+ExecStartPre=-/usr/bin/sleep 30
```

Comment:
Should we align this wait time with whatever we value we decide for NVMe IO timeouts? https://github.com/bottlerocket-os/bottlerocket/issues/2758. The maximum possible value of 4294967295 for that seems too large for this.